dashboard-frontend/src/pages/StudyPage/ParticipantsAcquiredPerSegmentTable.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import Table, { TableSkeleton } from './Table';
+import useStudy from './useStudy';
+import { formatNumber, parseNumber } from '../../helpers/numbers';
+import { StudySegmentProgressResource } from '../../types/study';
+
+const segmentsPerPage = 7;
```

Comment:
I would increase this quite a bit, the table can go down the page there's nothing below it anyways!

Reply:
Yes, it can be increased a bit.

The idea is that the table with its pagination controls fit vertically in the most used screen sizes, to not harm the user experience.

After checking https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide to obtain the most used screen sizes worldwide and trying multiple variations, I ended up incrementing it to 10 Segments per page.

Reply:
Yes, it can be increased a bit.

The idea is that the table with its pagination controls fit vertically in the most used screen sizes, to not harm the user experience.

After checking [https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide](https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide) to obtain the most used screen sizes worldwide and trying multiple variations, I ended up incrementing it to 10 Segments per page.